appserver/payara-appserver-modules/notification-newrelic-core/src/main/java/fish/payara/notification/newrelic/NewRelicNotificationConfigurer.java

```diff
@@ -0,0 +1,97 @@
+ * obtain a copy of the License at
+ * https://github.com/payara/Payara/blob/master/LICENSE.txt
+ * See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing the software, include this License Header Notice in each
+ * file and include the License file at glassfish/legal/LICENSE.txt.
+ *
+ * GPL Classpath Exception:
+ * The Payara Foundation designates this particular file as subject to the "Classpath"
+ * exception as provided by the Payara Foundation in the GPL Version 2 section of the License
+ * file that accompanied this code.
+ *
+ * Modifications:
+ * If applicable, add the following below the License Header, with the fields
+ * enclosed by brackets [] replaced by your own identifying information:
+ * "Portions Copyright [year] [name of copyright owner]"
+ *
+ * Contributor(s):
+ * If you wish your version of this file to be governed by only the CDDL or
+ * only the GPL Version 2, indicate your decision by adding "[Contributor]
+ * elects to include this software in this distribution under the [CDDL or GPL
+ * Version 2] license."  If you don't indicate a single choice of license, a
+ * recipient has the option to distribute your version of this file under
+ * either the CDDL, the GPL Version 2 or to extend the choice of license to
+ * its licensees as provided above.  However, if you add GPL Version 2 code
+ * and therefore, elected the GPL Version 2 license, then the option applies
+ * only if the new code is made subject to such option by the copyright
+ * holder.
+ */
+package fish.payara.notification.newrelic;
+
+import com.google.common.base.Strings;
+import fish.payara.nucleus.notification.admin.BaseNotificationConfigurer;
+import fish.payara.nucleus.notification.configuration.NotificationServiceConfiguration;
+import org.glassfish.api.Param;
+import org.glassfish.api.admin.*;
+import org.glassfish.config.support.CommandTarget;
+import org.glassfish.config.support.TargetType;
+import org.glassfish.hk2.api.PerLookup;
+import org.jvnet.hk2.annotations.Service;
+
+import java.beans.PropertyVetoException;
+
+/**
+ * @author mertcaliskan
+ */
+@Service(name = "notification-newrelic-configure")
+@PerLookup
+@CommandLock(CommandLock.LockType.NONE)
+@ExecuteOn({RuntimeType.DAS, RuntimeType.INSTANCE})
+@TargetType(value = {CommandTarget.DAS, CommandTarget.STANDALONE_INSTANCE, CommandTarget.CLUSTER, CommandTarget.CLUSTERED_INSTANCE, CommandTarget.CONFIG})
+@RestEndpoints({
+        @RestEndpoint(configBean = NotificationServiceConfiguration.class,
+                opType = RestEndpoint.OpType.POST,
+                path = "notification-newrelic-configure",
+                description = "Configures New Relic Notification Service")
+})
+public class NewRelicNotificationConfigurer extends BaseNotificationConfigurer<NewRelicNotifierConfiguration, NewRelicNotifierService> {
+
+    @Param(name = "key")
+    private String key;
+
+    @Param(name = "accountId")
```

Comment:
You can use the _alias_ Param option to specify lowercase as an acceptable alternative:

`@Param(name = "accountId", alias = "accountid")`